src/component/feature/FeatureToggleList/FeatureToggleListTable.tsx

```diff
@@ -207,14 +238,46 @@ export const FeatureToggleListTable: VFC = () => {
             <SearchHighlightProvider value={globalFilter}>
                 <Table {...getTableProps()}>
                     {/* @ts-expect-error -- fix in react-table v8 */}
-                    <SortableTableHeader headerGroups={headerGroups} />
-                    <TableBody {...getTableBodyProps()}>
-                        {rows.map(row => {
+                    <SortableTableHeader headerGroups={headerGroups} flex />
+                    <TableBody
+                        {...getTableBodyProps()}
+                        style={{
+                            height: `${rowHeight * rows.length}px`,
+                            position: 'relative',
+                        }}
+                    >
+                        {rows.map((row, i) => {
```

Comment:
Let's use the full names for parameters, vite will minify for us anyway: `(row, index)`

Reply:
Fixed. 

`i` is so universal, that I didn't even thing about it. `e` can stand for Event or Error, and I don't like weird things like `err` or `evt`. I think we have `i` in other places.